app/frontend/containers/EmailTargetView/EmailTargetView.js

```diff
@@ -0,0 +1,211 @@
+// @flow
+import React, { Component } from 'react';
+import _ from 'lodash';
```

Comment:
Importing individual lodash functions reduces bundle size quite a bit. In this file you're only using `find`so you can import only it: 

```js
import find from 'lodash/find';
```